pkg/kube/apisix/apis/config/v2beta1/types.go

```diff
@@ -53,14 +55,14 @@ type ApisixRouteHTTP struct {
 	Priority int                  `json:"priority,omitempty" yaml:"priority,omitempty"`
 	Match    ApisixRouteHTTPMatch `json:"match,omitempty" yaml:"match,omitempty"`
 	// Deprecated: Backend will be removed in the future, use Backends instead.
-	Backend ApisixRouteHTTPBackend `json:"backend" yaml:"backend"`
+	Backend v2alpha1.ApisixRouteHTTPBackend `json:"backend" yaml:"backend"`
```

Comment:
Looks strange, if we use data structures in `v2alpha1`. Why not just implement them here.

Reply:
Just for reusing the structures and the translation logic will be reused too.